
# **Molecular Phylogenetics, Monash University Malaysia, Kuala Lumpur**

Teacher Niklas Wahlberg

## **Introduction**

In this course we will learn in practice how to download data from GenBank, create a multigene dataset, and analyse the dataset using maximum likelihood. We will also hear about how we can use museum specimens for molecular analyses, in particular whole genome sequencing.

## **Schedule**

**Location:** Monash University Malaysia, Kuala Lumpur

<!-- -->

| Hours\Days | November 12 |
| --- | ---- |
| 9:00-9:30 | Registration |
| 9:30-10:30 | Presentation: Molecular phylogenetics |
| 10:30-11:00 | Break |
| 11:00-12:30 | Tutorials: making a phylogeny  |
| 12:30-14:00 | Lunch |
| 14:00-14:45 | Presentation: Museomics |
| 14:45-15:00 | Short break |
| 15:00-16:30 | Tutorials continue |

<!--  

| 9:30-11:00 | - | Lecture 2: Understanding trees and DNA substitution models (JR) | Lecture 3: Properties of DNA data and assessing robustness of hypotheses (JR) | Museomics workshop (NW) |
| 11:00-12:00 | - | Tutorial 1 | Tutorial 3 (JR, NW) | - |
| 12:00-14:30 | Lunch | Lunch | Lunch | - |
| 14:30-16:30 | Lecture 1: Introduction (NW) | Tutorial 2 (JR, NW) | Tutorial 4 (JR, NW) | - |

| Hours\Days | March 26 | March 27 | March 28 | March 29 |
| --- | ---- | ---- | ---- | ---- |
| 9:00-10:00 | - | Tutorial 1 (JR, NW) | Tutorial 3 (JR, NW) | Lecture 6: Timing of Divergence (NW) |
| 10:00-12:00 | - | Tutorial 2 (JR, NW) | Tutorial 4 (JR, NW) | Museomics workshop (NW) |
| 12:00-13:00 | Lunch | Lunch | Lunch | Lunch |
| 13:00-14:30 | Lecture 1: Introduction (NW) | Lecture 3: Introduction to model based methods (JR) | Lecture 5: Properties of DNA data and assessing robustness of hypotheses (JR) | - |
| 14:30-16:00 | Lecture 2: Understanding trees and alignments (NW) | Lecture 4: Statistical frameworks for modelling in phylogenetics (ML and Bayesian) (JR) | Tutorial 5 (JR, NW) | - |
-->
<!--  -->




## **Index**

### [Course slides](./Lectures/)

Here we will upload all the files and information shared during the lectures!


### [Data used in the course](./Data/)

Here you will find the files we create during the course. (If at some point you are at a loss because your attempts to produce the results for the next step failed, here you can find the files you will need so you can keep up with the class.)


### **Tutorials**

Here are deposited the instructions for the assignments and exercises.


 (1) [Learn to generate datasets](./Tutorials/1.DatasetManipulation/)
	
 (2) [Align the gene markers](./Tutorials/2.Alignments/)
 
 (3) [Select the best evolutionary model and partitioning scheme](./Tutorials/3.ModelSelection/)
 
 (4) <!--[Bayesian methods](./Tutorials/4.BayesianInference/) -->[Maximum likelihood methods](./Tutorials/5.MaximumLikelihood/)
 
<!--
 (6) [Timing of divergence analysis](./Tutorials/6.TimingDivergence/)
 
 (7) [Tree manipulation](./Tutorials/7.TreeManipulation/)
 
 (8) [Diversification analysis](./Tutorials/8.Diversification/)
 
 (9) [Gene tree/species tree analysis](./Tutorials/11.ASTRAL/)
 
 (10) [Historical biogeography](./Tutorials/10.Biogeography/)
 -->



### **Programs used in the tutorials**

Here you can find a list of the programs we are going to use and links to where you can download them.
 
 * **Text Editors:**
   	- **TextWrangler** (older versions) or **BBEdit** for Mac users
   	- **Notepad++** for Windows users
   	- **Linux** users not covered

 <!-- * [MAFFT](https://mafft.cbrc.jp/alignment/software/) -->

 * [AliView](http://www.ormbunkar.se/aliview/downloads/)

 * [IQ-TREE2](http://www.iqtree.org/)
 

	
 * [FigTree](http://tree.bio.ed.ac.uk/software/figtree/)

<!--
 * [MrBayes](http://nbisweden.github.io/MrBayes/)
 * [Tracer](https://github.com/beast-dev/tracer/releases/tag/v1.7.1)
 
 * [BEAST2](http://www.beast2.org/)
 
 * [R](https://www.r-project.org/)
 
 * [ASTRAL](https://github.com/smirarab/ASTRAL)
-->

